jestSetup.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line n/no-unpublished-require
+require('jest-fetch-mock').enableMocks()
```

Comment:
Without this package and setup, the `Request` and `Headers` objects aren't found in the context of the test file that was added (`request.spec.ts`)